common/test/acceptance/tests/studio/test_studio_settings.py

```diff
@@ -510,6 +511,15 @@ def test_studio_settings_page_a11y(self):
             ],
         })
 
+        # TODO: Figure out how to get CodeMirror to pass accessibility testing
+        # We use the CodeMirror Javascript library to
+        # add code editing to a number of textarea elements
+        # on this page. CodeMirror generates markup that does
+        # not pass our accessibility testing rules.
+        self.settings_page.a11y_audit.config.set_scope(
+            exclude=['textarea']
+        )
```

Comment:
@clrux Can I use any CSS selector here to tighten the scope of the exclude or do I need to exclude all textarea elements?